server/src/config/database.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable class-methods-use-this */
+import mongoose from 'mongoose';
+
+class Database {
```

Comment:
why class? when it is just an object?

Reply:
Makes sense. I started off with different thought and it ended up being a single object.
Will fix.

Reply:
Done.